pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -57,7 +58,7 @@ type MasterConfig struct {
 	CloudProvider     cloudprovider.Interface
 }
 
-func BuildKubernetesMasterConfig(options configapi.MasterConfig, requestContextMapper kapi.RequestContextMapper, kubeClient *kclient.Client, projectCache *projectcache.ProjectCache) (*MasterConfig, error) {
+func BuildKubernetesMasterConfig(options configapi.MasterConfig, requestContextMapper kapi.RequestContextMapper, kubeClient *kclient.Client, projectCache *projectcache.ProjectCache, authorizer authorizer.Authorizer) (*MasterConfig, error) {
```

Comment:
This is starting to look leaky.  Why shouldn't we pass in a `PluginInitializer`?